app/services/item_service.rb

```diff
@@ -0,0 +1,43 @@
+class ItemService
+  class << self
+    def create_item_from_upc(upc)
+      data = Edamam.get_nutrition_data_from_upc(upc)
+      return unless data
+
+      health_label_ids = HealthLabel
+        .ids
+        .values_at(*data[:health_labels].map { |health_label| health_label.downcase.to_sym })
+        .compact
+
+      custom_measurement_unit_names = data[:measurement_units] - MeasurementUnit.standard.keys
+      custom_measurement_units = MeasurementUnit.where(name: custom_measurement_unit_names).to_a
```

Comment:
Why not `MeasurementUnit.custom` to match `standard`?

Reply:
I could do that. That would have to be an actual scope method though. This is just a cached array.